net/usbip/Makefile

```diff
@@ -82,13 +82,13 @@ endef
 CONFIGURE_VARS+= $(if $(CONFIG_USE_GLIBC),LIBS='-lbsd -lpthread')
 CFLAGS+="$(TARGET_CFLAGS) -I$(STAGING_DIR)/usr/include"
 
-USB_IDS_REV:=0a6be488b4c1485d30c9efee1cc895af5d4a1383
+USB_IDS_REV:=016be785fd4f5f895c6482ec484507d5a08839fa
 USB_IDS_FILE:=usb.ids.$(USB_IDS_REV)
 define Download/usb.ids
   FILE:=$(USB_IDS_FILE)
   URL_FILE:=usb.ids
-  URL:=@GITHUB/gentoo/hwids/$(USB_IDS_REV)
-  HASH:=2d0403ffccc0ff7150e32c0bf8e2878816b76ba08d2c3bb1059f33627a3da30f
+  URL:=@GITHUB/vcrhonek/hwdata/$(USB_IDS_REV)
+  HASH:=b1877a9fd30c2f9dedec1657963775e7f261c892eb7d6f4281c453bd73492489
```

Comment:
URL http://www.linux-usb.org/usb.ids should be better than files from random github repos.

Reply:
https://github.com/openwrt/openwrt/commit/f23271f3b4f54d1e5ddf7b85786f9a30f38409ad I am based on this commits

Reply:
Is it feasible to make usb.ids a separate package in the base system and have usbip select it?  Better if `FILE` can have revision date in it.

Reply:
This idea is good, but I think need to modify other packages? If someone creates this package and keeps updating it...Otherwise, the status quo should be maintained. (Sorry, I use google translation